.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+*Before* submitting a pull request, please make sure the following is done...
+
+1. Fork the repo and create your branch from `master`.
```

Comment:
For example, If a user is not the first-time contributor, he can create a branch based on his old work. And there are many cases like this.